common/reportMetrics.py

```diff
@@ -432,15 +442,25 @@ def process_replay(file_path):
                                     max_timestamp_epoch = long(row[i])
                             else:
                                 column_name = field_names[i].strip()
-                                metric = column_name.split("[")[0]
+                                metric_key = column_name.split("[")[0]
                                 if column_name.find("]") == -1:
                                     column_name = column_name + "[-]"
+
                                 if column_name.find(":") == -1:
-                                    group_id = get_grouping_id(metric, grouping_map)
+                                    print metric_key
```

Comment:
please use logger.info() rather than print

Reply:
Removed that print statement. Not useful.